app/views/sessions/new2.html.erb

```diff
@@ -0,0 +1,44 @@
+<head>
+		<meta name="author" content="Matthew Lewicki">
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+		<meta charset="UTF-8">
+		<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+		<title>Spotify Login</title>
+	</head>
+	<body>
+    <% if flash.now[:danger] %>
+      <div class="alert alert-danger "><%=  flash.now[:danger] %></div>
+    <% end %>
+		<section>
+       <h1>Welcome to Tunes and Food</h1>
+			 <p>Log in with your Spotify account </p>
+
+       <%= form_for :session, url: sign_in_path, class: "form" do |f| %>
+        <div class="spotify_logo"><img src="https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcRNITBCIzPtexvfXQbi3DG4nsDjtebT0jUth_uWneg7HxCCHEcI-hnrU5gY"></div>
+          <fieldset>
+            <% f.email_field :email, :placeholder => "Email" %>
+          </fieldset>
+          <fieldset>
+            <% f.password_field :password, :placeholder => "Password" %>
+          </fieldset>
+          <fieldset>
+            <%= f.submit "Log in" %>
+          </fieldset>
+       <% end %>
+			 <div class="container">
+    <h1>Search for an Artist</h1>
+    <p>Type an artist name and click on "Search". Then, click on any album from the results to play 30 seconds of its first track.</p>
+    <form id="search-form">
+        <input type="text" id="query" value="" class="form-control" placeholder="Type an Artist Name"/>
+        <input type="submit" id="search" class="btn btn-primary" value="Search" />
+    </form>
+    <div id="results"></div>
+</div>
+<script id="results-template" type="text/x-handlebars-template">
+    {{#each albums.items}}
+    <div style="background-image:url({{images.0.url}})" data-album-id="{{id}}" class="cover"></div>
```

Comment:
Thank you so much for taking the time to review my code!!! I really
appreciate your help.

Yordanos

On Tue, May 24, 2016 at 3:46 PM, CShekta <notifications@github.com> wrote:

> In app/views/sessions/new2.html.erb
> <https://github.com/Ada-C5/tunes-takeout/pull/20#discussion_r64488571>:
>
> > +          <fieldset>
> > +            <%= f.submit "Log in" %>
> > +          </fieldset>
> > +       <% end %>
> > +			 <div class="container">
> > +    <h1>Search for an Artist</h1>
> > +    <p>Type an artist name and click on "Search". Then, click on any album from the results to play 30 seconds of its first track.</p>
> > +    <form id="search-form">
> > +        <input type="text" id="query" value="" class="form-control" placeholder="Type an Artist Name"/>
> > +        <input type="submit" id="search" class="btn btn-primary" value="Search" />
> > +    </form>
> > +    <div id="results"></div>
> > +</div>
> > +<script id="results-template" type="text/x-handlebars-template">
> > +    {{#each albums.items}}
> > +    <div style="background-image:url({{images.0.url}})" data-album-id="{{id}}" class="cover"></div>
>
> cOoL!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ada-C5/tunes-takeout/pull/20/files/09c8be8953be05370d120eba861c83a440955ec1#r64488571>
>